### DISCUSSION TITLE
Custom Error and Empty states for Table

### DISCUSSION THREAD

**Discussion by krishna-sharma-dev** [heroui-inc/heroui#693] [Help]
Hello, 
Wanted to know how can i add a custom Error or Empty States inside the table if there is no data or in case of an error.
I would still want the table outline to be showed but the error/empty states(icon) to be shown inside the table.
Something like this (taken from antd):
<img width="949" alt="Screenshot 2022-08-26 at 9 44 23 PM" src="https://user-images.githubusercontent.com/110056286/186948404-893dde47-c60e-4b60-8801-8d96f5fd0aa3.png">

**Comment by krishna-sharma-dev** [comment]
Any update on this yet please?

**Comment by krishna-sharma-dev** [comment]
Changed the category to "help" just to see if i am missing something?

**Comment by krishna-sharma-dev** [comment]
Still waiting for this.. Any help would be appreciated!

**Comment by jrgarciadev** [comment]
Hey please update to v2.0 https://nextui.org/blog/nextui-v2 due to v1 will not receive more updates.

https://nextui.org/docs/components/table#empty-state
